js/kanban.js

```diff
@@ -699,36 +849,55 @@
 
          $(self.element + ' .kanban-container').on('submit', '.kanban-add-form', function(e) {
             e.preventDefault();
-            var form = $(e.target);
-            var data = {};
-            data['inputs'] = form.serialize();
-            data['itemtype'] = form.prop('id').split('_')[2];
-            data['action'] = 'add_item';
+            const form = $(e.target);
+            const data = {
+               inputs: form.serialize(),
+               itemtype: form.prop('id').split('_')[2],
+               action: 'add_item'
+            };
 
             $.ajax({
                method: 'POST',
-               //async: false,
                url: (self.ajax_root + "kanban.php"),
                data: data
             }).done(function() {
                self.refresh();
             });
          });
+
+         $(self.element + ' .kanban-container').on('click', '.kanban-item .kanban-item-title', function(e) {
+            e.preventDefault();
+            const card = $(e.target).closest('.kanban-item');
+            const [itemtype, items_id] = card.prop('id').split('-');
+            if ($('#kanban-dialog').length === 0) {
+               $(self.element).append('<div id="kanban-dialog"></div>');
+               // After initializing the dialog, it gets moved automatically outside the Kanban container. That's why it has an ID instead of a class.
+               $(self.element + ' #kanban-dialog').dialog({
+                  autoOpen: false,
+                  modal: true,
+                  resizable: true,
+                  draggable: true,
+                  height: 700,
+                  width: 800
+               });
+            }
+            $('#kanban-dialog').load((self.ajax_root + "kanban.php/?action=show_card_edit_form&itemtype="+itemtype+"&card=" + items_id)).dialog("open");
```

Comment:
this `/` at the end of the url do 404 errors on my current setup.

Is it mandatory ?
If i remove it, it seems to be ok on my side.

Reply:
It should be optional to have it in a valid URL. Works for me with or without, so I removed the slash.